CarCRUD.cs

```diff
@@ -106,5 +104,20 @@ public void Update(string Id)
                 }
             }
         }
+        public Car ListAll ()
```

Comment:
Un detalle de estilos 
```suggestion
        public Car ListAll()
```